xarray/backends/api.py

```diff
@@ -394,7 +397,11 @@ def open_dataarray(*args, **kwargs):
     --------
     open_dataset
     """
-    dataset = open_dataset(*args, **kwargs)
+    dataset = open_dataset(filename_or_obj, group=None, decode_cf=True,
```

Comment:
You need to pass in the values from the `open_dataarray` call, not the defaults: `open_dataset(filename_or_obj, group=group, decode_cf=decode_cf, ...)`.  As written, none of the values other than `filename_or_obj` that the user specifies will get used: `mask_and_scale` will always be `True`, etc.